### PR TITLE
add workflow api

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Cancel the running of a job
```
Comment (whitewum) [inline docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md:1]: Cancel a running job

**Review by whitewum — APPROVED** [review]

File: docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
+# Cancel a running job
+
+This topic describes how to use API to cancel a running job.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md:3]: use API ---> use an AP

File: docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+|Parameters|Type|If required|Default value|Example| Description|
+|:---|:---|:---|:---|:---|:---|
+|`job_id`|number|yes|-|`1964`|The job ID. It can be queried through the API [Get a list of all jobs](api-get-jobs.md) or viewed on the job list page.|
+
+### Headers parameters
+
+|Parameters|Type|If required|Default value|Example| Description|
+|:---|:---|:---|:---|:---|:---|
+|`Content-Type`|string|yes|-|`application/x-www-form-urlencoded`|The content type.|
+|`explorer_token`|string|yes|-|`eyJhbxxx`|The authorization token that used to verify account information. For details, see [Workflow API overview](workflow-api-overview.md).|
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md:24]: that used ----> that is used / used

File: docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+
+```bash
+curl -i -X PUT -H "Content-Type: application/x-www-form-urlencoded" -H "Cookie: "explorer_token=eyJhbxxx"" http://192.168.8.145:7002/api-open/v1/jobs/30600/cancel
+```
+
+## Response parameters
+
+|Parameters|Type|Example|Description|
+|:---|:---|:---|:---|
+|`code`    | number | `0`       |  The result code of the request. Return `0` if the request is successful, and return the error code if the request is unsuccessful. For details, see [Workflow API overview](workflow-api-overview.md).            |
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/workflow-api/api-cancel-job.md:40]: return the error code --> return an error code

**Review by abby-cyber — DISMISSED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by randomJoe211 — APPROVED** [review]

**Pull request merged**